initfiles/componentfiles/thor/run_thor

```diff
@@ -35,7 +34,7 @@ echo $$ > $RUN_THOR_PID_NAME
 export SENTINEL="thor.sentinel"
 while [ 1 ]; do
     # update slaves file in case state of environment has been altered since last run
-    daliadmin $DALISERVER dfsgroup ${groupName} slaves
+    daliadmin server=$DALISERVER dfsgroup ${groupName} slaves
```

Comment:
I see now that it's not just treating the 1st param specially, but only if leading digit or "."
server= definitely better.

But there are a couple of other places that call daliadmin that would be worth changing similarly at the same time, namely in stop_thor and start_backupnode